instrumentation-core/spring/spring-boot-autoconfigure/README.md

```diff
@@ -0,0 +1,428 @@
+    <artifactId>opentelemetry-api</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+
+   <!-- simple span exporter -->
+   <!-- outputs spans to console -->
+   <!-- provides opentelemetry-sdk artifact -->
+   <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporters-logging</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+
+</dependencies>
+```
+
+For Gradle add to your dependencies:
+
+```groovy
+//opentelemetry spring auto-configuration
+implementation 'io.opentelemetry.instrumentation:opentelemetry-spring-boot-autoconfigure:OPENTELEMETRY_VERSION'
+//opentelemetry
+implementation 'io.opentelemetry:opentelemetry-api:OPENTELEMETRY_VERSION'
+//opentelemetry exporter
+implementation 'io.opentelemetry:opentelemetry-exporters-otlp:OPENTELEMETRY_VERSION'
+```
+
+### Features
+
+#### Dependencies
+
+The following dependencies are optional but are required to use the corresponding features.
+
+Replace `SPRING_VERSION` with the version of spring you're using.
+ - Minimum version: `3.1`
+
+Replace `SPRING_WEBFLUX_VERSION` with the version of spring-webflux you're using.
+ - Minimum version: `5.0`
+
+Replace `SLF4J_VERSION` with the version of slf4j you're using.
+
+For Maven add to your `pom.xml`:
+
+```xml
+<dependencies>
+  <!-- opentelemetry exporters-->
+  <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporters-jaeger</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+  <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporters-zipkin</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+  <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporters-logging</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+
+  <!-- Used to autoconfigure spring-web -->
+  <dependency>
+    <groupId>org.springframework</groupId>
+    <artifactId>spring-web</artifactId>
+    <version>SPRING_VERSION</version>
+  </dependency>
+
+  <!-- Used to autoconfigure spring-webmvc -->
+  <dependency>
+    <groupId>org.springframework</groupId>
+    <artifactId>spring-webmvc</artifactId>
+    <version>SPRING_VERSION</version>
+  </dependency>
+
+  <!-- Used to autoconfigure spring-webflux -->
+  <dependency>
+    <groupId>org.springframework</groupId>
+    <artifactId>spring-webflux</artifactId>
+    <version>SPRING_WEBFLUX_VERSION</version>
+  </dependency>
+
+  <!-- Used to enable instrumentation using @WithSpan  -->
+  <dependency>
+    <groupId>org.springframework</groupId>
+    <artifactId>spring-aop</artifactId>
+    <version>SPRING_VERSION</version>
+  </dependency>
+  <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-extension-auto-annotations</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+
+  <!-- Slf4j log correlation support -->
```

Comment:
we have this?

Reply:
Not yet. This work is blocked by this issue: https://github.com/open-telemetry/opentelemetry-java/issues/922. I left it in as a placeholder. But since logging support is not required for GA I don't think this issue will be resolve in the next few weeks. I'll remove this for now.